x-pack/plugins/security_solution/public/resolver/view/submenu.tsx

```diff
@@ -52,73 +43,46 @@ interface ResolverSubmenuOption {
 
 export type ResolverSubmenuOptionList = ResolverSubmenuOption[] | string;
 
-const OptionListItem = styled.div`
-  width: 175px;
-`;
-
-const OptionList = React.memo(
+/**
+ * This will be the "host button" that displays the "total number of related events" and opens
+ * the sumbmenu (with counts by category) when clicked.
+ */
+const DetailHostButton = React.memo(
```

Comment:
`AggregateEventCountButton` or maybe `RelatedEventsCountButton` or something of that ilk? I feel like the terms `detail` and `host` could be a bit misleading and don't really tie in to RelatedEvents

Reply:
This is meant to be a little more generic (it preserves some DNA from a past version that even allows things that aren't a menu but just trigger an action). I agree that "Host" could be confusing in this context, though. How about something like "SubButton"?

Reply:
Yea, that works

Reply:
Pushed - thanks!